scripts/ci/install_node_modules.sh

```diff
@@ -5,5 +5,4 @@ set -x #echo on
 cd bokehjs
 npm install --location=global npm
 npm ci --no-progress
-node make examples --no-build
-git restore package*.json # avoid .dirty in the package version
+node make --no-up-to-date examples:build
```

Comment:
This now fails. Do we expect bokehjs to be already built at this point or not? `--no-build` didn't do anything for awhile now, so bokehjs was built here regardless.

Reply:
It looks like this was unnecessary altogether.